src/api/core/warren/utils.py

```diff
@@ -16,3 +30,32 @@ def pipe(*functions: Callable) -> Callable:
 
     """
     return reduce(lambda f, g: lambda x: g(f(x)), functions, lambda x: x)
+
+
+def get_token(token: Annotated[str, Depends(oauth2_scheme)]):
```

Comment:
Maybe we should be more specific?

```suggestion
def get_lti_token(token: Annotated[str, Depends(oauth2_scheme)]) -> LTIToken:
```